CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+# Contributing
+
+We accept contributions via Pull Requests on [GitHub](https://github.com/Yoast/YoastSEO.js).
+
+## Pull Requests
+
+- **Coding standards** - The easiest way to verify the coding standards is to use `grunt eslint`.
+
+- **Add tests!** - Your patch won't be accepted if it doesn't have tests.
+
+- **Document any change in behaviour** - Make sure the `README.md` and any other relevant documentation are kept up-to-date.
+
+- **Create feature branches** - Don't ask us to pull from your master branch.
+
+- **One pull request per feature** - If you want to do more than one thing, send multiple pull requests.
+
+- **Send coherent history** - Make sure each individual commit in your pull request is meaningful. If you had to make multiple intermediate commits while developing, please [squash them](http://www.git-scm.com/book/en/v2/Git-Tools-Rewriting-History#Changing-Multiple-Commit-Messages) before submitting.
```

Comment:
I would not include this requirement. I would be VERY happy to accept a PR with a nice fix regardless if it took 15 "ugly" commits to get there. I would much rather see a request for well documented PR's instead.